src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -109,9 +115,29 @@ public CitationRelationsTab(DialogService dialogService,
 
         this.entryTypesManager = bibEntryTypesManager;
         this.duplicateCheck = new DuplicateCheck(entryTypesManager);
-        this.bibEntryRelationsRepository = new BibEntryRelationsRepository(new SemanticScholarFetcher(preferences.getImporterPreferences()),
-                new BibEntryRelationsCache());
-        citationsRelationsTabViewModel = new CitationsRelationsTabViewModel(databaseContext, preferences, undoManager, stateManager, dialogService, fileUpdateMonitor, taskExecutor);
+
+        try {
+            var jabRefPath = Paths.get("/home/sacha/Documents/projects/JabRef");
+            var citationsPath = Path.of(jabRefPath.toAbsolutePath() + File.separator + "citations");
+            var relationsPath = Path.of(jabRefPath.toAbsolutePath() + File.separator + "references");
```

Comment:
What about `Path.resolve()`? I think it's more idiomatic. What if `toAbsolutePath()` will return a trailing slash? That is why there is a dedicated `resolve()`.